client/src/App.js

```diff
@@ -1,18 +1,34 @@
-import React, { Component } from "react";
-import { BrowserRouter, Route, Switch, Redirect } from "react-router-dom";
-import "./App.css";
+import React, { Component } from 'react';
```

Comment:
don't commit changes on files outside your component unless it's related to your PR directly

Reply:
ok,
thanks